root/static/scripts/guess-case/modes.js

```diff
@@ -8,7 +8,7 @@ const {assign} = require('lodash');
 
 const {l} = require('../common/i18n');
 const getBooleanCookie = require('../common/utility/getBooleanCookie');
-const {isPrepBracketWord, isPrepBracketSingleWord} = require('./utils');
+const {isPrepBracketWord, isPrepBracketSingleWord, turkicUpperCase, turkicLowerCase} = require('./utils');
```

Comment:
Ah. Still, it isn’t a feature of the Turkic languages, just of some writing systems (Turkish originally, which then spread to neighbouring countries with related languages). I would prefer not to use Turkic here; in the same way as I wouldn’t like English mode to be called “Germanic mode” even in case Norwegian also used it.